src/include/optimizer/operators.h

```diff
@@ -141,17 +142,6 @@ class LogicalGroupBy : public OperatorNode<LogicalGroupBy> {
 };
```

Comment:
We have came to the conclusion that if we need to enumerate the implementation for a component, e.g. hash group by v.s. sort group by, index scan v.s. seq scan, then we may want treat it as a operator. If the implementation is fixed for a component, e.g. order by, then we will store it as a property.